Python-tp2-pendu/pack/fonctions.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3.6
+# -*-coding:utf-8 -*
+
+"""Pendu Game functions"""
+
+import pickle
+
+
+def scores_checking():
+    """Checking and returning if an history of scores is already existing or create it"""
+    try:
+        players_history = open("pack/scores", "rb")
+    except FileNotFoundError:
+        players_history = open("pack/scores", "wb")
+        scores_dict = dict()
+        save = pickle.Pickler(players_history)
+        save.dump(scores_dict)
+    else:
+        scores_history = pickle.Unpickler(players_history)
+        scores_dict = scores_history.load()
+    finally:
+        players_history.close()
+    return scores_dict
+
+
+def gamer_history(scores_dict):
+    """Checking if the gamer have already a score and create one if not"""
+    print("Saisissez votre nom de joureur : ")
+    answer = input()
```

Comment:
Tu peux mettre ton texte directement dans le `input()` plutôt que de faire un `print()` puis un `input()`:
```suggestion
    answer = input("Saisissez votre nom de joueur : ")
```

Reply:
Modifié. Merci.